### PR TITLE
Send T-0 email

### PR DESCRIPTION
**Pull request by patrickvenetz**
This Pull Request sends an email on the very day a membership ends. Email is scheduled for 6:30 in the morning.

bildschirmfoto 2019-01-13 um 19 00 29

Added `sequenceNumber`, `cancel_url` and `grace_end_date` into template variables. Latter describes when an account is scheduled to be deactivated.

### CODE REVIEW

File: servers/republik/modules/crowdfundings/lib/Mail.js [hunk 1]
```diff
@@ -595,14 +605,23 @@ mail.prepareMembershipOwnerNotice = async ({ user, endDate, cancelUntilDate, tem
...
+        content: formattedGraceEndDate
+      },
       { name: 'days_left',
         content: daysLeft
       },
       { name: 'cancel_until_date',
         content: dateFormat(cancelUntilDate)
+      },
+      { name: 'sequence_number',
+        content: sequenceNumber
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/Mail.js]: why is it not the sequenceNumber of `user.memberships[0].sequenceNumber`? Why are the `membershipId` and `sequenceNumber` not dependent on each other?

File: servers/republik/modules/crowdfundings/lib/scheduler/owners.js [hunk 2]
```diff
@@ -165,10 +167,11 @@ const inform = async (args, context) => {
         const templatePayload = await context.mail.prepareMembershipOwnerNotice({
           user,
           endDate: prolongBeforeDate,
-          cancelUntilDate: moment(prolongBeforeDate)
-            .subtract(2, 'days'),
+          cancelUntilDate: moment(prolongBeforeDate).subtract(2, 'days'),
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/scheduler/owners.js]: `cancelUntilDate` will always be in the past for `membership_owner_prolong_notice_0` right?

File: servers/republik/modules/crowdfundings/lib/scheduler/owners.js [hunk 3]
```diff
@@ -165,10 +167,11 @@ const inform = async (args, context) => {
...
+          graceEndDate: moment(prolongBeforeDate).add(14, 'days'),
           templateName: bucket.templateName
         }, context)
+        console.log(templatePayload)
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/scheduler/owners.js]:
```suggestion
```

File: servers/republik/modules/crowdfundings/lib/translations.json [hunk 4]
```diff
@@ -479,8 +479,8 @@
       "value": "Möchten Sie Ihre Abo verlängern oder die Republik verlassen? Entscheiden Sie sich bis {endDate}"
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/translations.json]:
```suggestion
      "value": "Möchten Sie Ihr Abo verlängern oder die Republik verlassen? Entscheiden Sie sich bis {endDate}"
```
typo (fix in gsheet)

File: servers/republik/modules/crowdfundings/lib/translations.json [hunk 5]
```diff
@@ -479,8 +479,8 @@
...
     },
     {
-      "key": "api/email/membership_owner_prolong_notice_2/subject",
-      "value": "Möchten Sie Ihre Abo verlängern oder die Republik verlassen? Entscheiden Sie sich bis {endDate} "
+      "key": "api/email/membership_owner_prolong_notice_0/subject",
+      "value": "Ihre Abonnementsrechnung (#{sequenceNumber})"
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/translations.json]: 🙄

File: servers/republik/modules/crowdfundings/lib/scheduler/owners.js (hunk 2 above)
Comment (patrickvenetz) [reply servers/republik/modules/crowdfundings/lib/scheduler/owners.js]: Yes.

File: servers/republik/modules/crowdfundings/lib/scheduler/owners.js [hunk 6]
```diff
@@ -72,16 +73,15 @@ const getBuckets = async ({ now }, { pgdb }) => {
       m."userId" != :PARKING_USER_ID
       AND m.active = true
       AND m.renew = true
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/scheduler/owners.js]: Please indicate with a big fat comment, that this only hold when a user only has one active membership

**Pull request merged**